pkg/migrate/cstor/volume.go

```diff
@@ -534,9 +534,16 @@ func (v *VolumeMigrator) updateStorageClass(pvName, scName string) error {
 			return err
```

Comment:
This [log](https://github.com/openebs/upgrade/blob/1e1e5181dd811f8e1dd3f247a29b91228afec805/pkg/migrate/cstor/volume.go#L528) should be either before `temporary sc creation` or just before creating CSI-SC. For each job having this log doesn't make sense if we multiple jobs upgrade sc.